app/models/df_w2_accessor.rb

```diff
@@ -0,0 +1,59 @@
+class DfW2Accessor
+  include DfXmlCrudMethods
+
+  SELECTORS = {
+    EmployeeSSN: 'EmployeeSSN',
+    EmployerEIN: 'EmployerEIN',
+    EmployerName: 'EmployerName BusinessNameLine1Txt',
+    EmployerStateIdNum: 'EmployerStateIdNum',
+    AddressLine1Txt: 'EmployerUSAddress AddressLine1Txt',
+    City: 'EmployerUSAddress CityNm',
+    State: 'EmployerUSAddress StateAbbreviationCd',
+    ZIP: 'EmployerUSAddress ZIPCd',
+    WagesAmt: 'WagesAmt',
+    AllocatedTipsAmt: 'AllocatedTipsAmt',
+    DependentCareBenefitsAmt: 'DependentCareBenefitsAmt',
+    NonqualifiedPlansAmt: 'NonqualifiedPlansAmt',
+    RetirementPlanInd: 'RetirementPlanInd',
+    ThirdPartySickPayInd: 'ThirdPartySickPayInd',
+    StateAbbreviationCd: 'W2StateTaxGrp StateAbbreviationCd',
+    StateWagesAmt: 'W2StateLocalTaxGrp W2StateTaxGrp StateWagesAmt',
+    StateIncomeTaxAmt: 'W2StateLocalTaxGrp W2StateTaxGrp StateIncomeTaxAmt',
+    LocalWagesAndTipsAmt: 'W2StateLocalTaxGrp W2StateTaxGrp W2LocalTaxGrp LocalWagesAndTipsAmt',
+    LocalIncomeTaxAmt: 'W2StateLocalTaxGrp W2StateTaxGrp W2LocalTaxGrp LocalIncomeTaxAmt',
+    LocalityNm: 'W2StateLocalTaxGrp W2StateTaxGrp W2LocalTaxGrp LocalityNm',
+    WithholdingAmt: 'WithholdingAmt',
+  }
+
+  attr_reader :node
+
+  def initialize(node = nil)
+    @node = if node
+              node
+            else
+              Nokogiri::XML(IrsApiService.df_return_sample).at('IRSW2')
```

Comment:
Also - it strikes me that if you do remove these default node values from the constructors, I believe the only difference between DfW2Accessor and Df1099rAccessor will be their SELECTORS constants. So IF YOU FELT LIKE IT, you could make an abstract superclass (i.e. one that raises NotImplementedError in its initialize method, because ruby is silly) that both classes inherit from and move everything except for SELECTORS into it.